modules/browser/BrowserCapabilities.js

```diff
@@ -219,6 +219,17 @@ export default class BrowserCapabilities extends BrowserDetection {
         return !this.isFirefox();
     }
 
+    /**
+     * Returns true if the browser supports track based statistics for the local video track. Otherwise,
+     * track resolution and framerate will be calculated based on the 'outbound-rtp' statistics.
+     * @returns {boolean}
+     */
+    supportsTrackBasedStats() {
```

Comment:
Currently we have WebRTC M111 on React Native, what stats should we be using?

Reply:
inbound-rtp based stats contain everything you need for a while (in Chromium-based browsers) already.

Reply:
Should have it since 2020 at least: https://source.chromium.org/chromium/chromium/src/+/main:third_party/webrtc/api/stats/rtcstats_objects.h;l=500;drc=b3e59696585d222a53c1c59292eb92a66e604643

Reply:
Nice!

Reply:
Yes, `outbound-rtp `have been available for quite a while now so it shouldn't affect the current React Native build. The check here is mostly for older Chrome versions (72) that do not generate `outbound-rtp` stats and for Safari that still doesn't support it. @fippo I checked the current stable for Firefox and see the stats, but will check again and update the check here.